packages/utilities/src/useFocusRects.ts

```diff
@@ -56,12 +56,14 @@ export const FocusRectsProvider = FocusRectsContext.Provider;
 /**
  * Initializes the logic which:
  *
- * 1. Subscribes keydown and mousedown events. (It will only do it once per window,
- *    so it's safe to call this method multiple times.)
- * 2. When the user presses directional keyboard keys, adds the 'ms-Fabric--isFocusVisible' classname
- *    to the document body, removes the 'ms-Fabric-isFocusHidden' classname.
- * 3. When the user clicks a mouse button, adds the 'ms-Fabric-isFocusHidden' classname to the
- *    document body, removes the 'ms-Fabric--isFocusVisible' classname.
+ * 1. Subscribes keydown and mousedown events. (It will only do it once for the outmost ThemeProvider/Fabric element in
```

Comment:
> ThemeProvider/Fabric [](http://example.com/codeflow?start=87&length=21)

This function isn't directly related to ThemeProvider/Fabric components so consider either removing this part or making it an "example" if you feel like there's value in having this info here

Reply:
Updated!